isotovideo

```diff
@@ -477,14 +509,17 @@ sub process_command {
         $current_test_full_name = $rsp->{full_name};
 
         # send debugging info for ws clients
-        myjsonrpc::send_json($cmd_srv_fd, {set_current_test => $current_test_name});
+        myjsonrpc::send_json($cmd_srv_fd, {
+                set_current_test       => $current_test_name,
+                current_test_full_name => $current_test_full_name,
+        });
         if ($pause_test_name
             && $current_test_name
             && $current_test_full_name
             && ($pause_test_name eq $current_test_name || $pause_test_name eq $current_test_full_name))
         {
-            print("isotovideo: pausing test execution of $pause_test_name\n");
-            $stop_passing_commands_to_backend = 1;
+            print("isotovideo: pausing test execution of $pause_test_name because we're supposed to pause there\n");
```

Comment:
Same as above, diag instead of print. On the other hand, I wonder if the reason to pause should be more clear.